Submitty/Submitty
# [Bugfix:TAGrading] Fix incorrect grading statistics

### What is the current behavior?
closes #6835 

### What is the new behavior?
All grading statistics are displayed correctly

### Other information?
To Test:
1. Make sure the maximum value on the histogram is always the real maximum overall score
2. make sure the "grade" buttons on the index page always display the correct score (if their is one)
